ZelBack/src/services/idService.js

```
@@ -279,7 +282,14 @@ async function verifyLogin(req, res) {
       const collection = config.database.local.collections.activeLoginPhrases;
       const query = { loginPhrase: message };
       const projection = {};
-      const result = await dbHelper.findOneInDatabase(database, collection, query, projection);
+      let result = await dbHelper.findOneInDatabase(database, collection, query, projection);
+      if (!result) {
+        const messageToSign = await getMessageToLogin(res, message);
+        if (!messageToSign) {
+          return;
```

there should be some throw instead of return I guess